### PR TITLE
Adding styles

### PR DESCRIPTION
**Pull request by mrnadaara**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,106 +4,159 @@
...
-        <section>
-          <button>Subscribe Now</button>
-          <button>Log In</button>
-        </section>
-      </nav>
-    </header>
+        </div>
+      </div>
+      <div class="right-nav">
+        <button>Log In</button>
```
Comment (whiz25) [inline index.html]: Make this button in line with the main logo 
Screenshot from 2019-09-23 22-26-43

File: index.html [hunk 2]
```diff
@@ -4,106 +4,159 @@
...
+        <button>Subscribe Now</button>
```
Comment (whiz25) [inline index.html]: Make this aligned on the same row as the logo.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job so far team :thumbsup:
However, we need to make the following changes before the project gets completed.
- [ ] I don't see your ```The Theory of Inflation``` section. This is required.
- [ ] Add the embedded video
- [ ] Add also the ```More in Space & Cosmos``` section
- [ ] Add a footer and links to other articles as well.

Make the changes and re-submit for another review.
## Just make your page look as close to the original page [here](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) as possible

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @mrnadaara and @Tekcoder   🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The **navbar** should have a **fixed CSS position**.

2. The **footer** also should have a **fixed CSS position**.

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Review by uche-inyama — APPROVED** [review]
### Nice, you have implemented the requested changes. :slightly_smiling_face: 

#### Go ahead and merge to master branch :rocket:

**Pull request merged**